queries.sql

```diff
@@ -81,3 +81,20 @@ SELECT species, AVG(escape_attempts) AS total_escape_attempts
 FROM animals
 WHERE date_of_birth BETWEEN '1990,01,01'::date AND '2000,12,31'
 GROUP By species; 
+
+
+--PROJECT 3
+-- What animals belong to Melody Pond?
+SELECT * FROM animals JOIN owners ON owners.id = animals.owner_id AND owners.fullname='Melody Pond'; 
+-- List of all animals that are pokemon (their type is Pokemon).
+SELECT * FROM animals JOIN species ON species.id = animals.species_id AND species.name='Pokemon';
+-- List all owners and their animals, remember to include those that don't own any animal.
+SELECT fullname , name From owners Left JOIN animals ON animals.owner_id=owners.id;
+-- How many animals are there per species?
+SELECT species.name , COUNT(animals.name) AS Animals_Count From species JOIN animals ON species.id=animals.species_id GROUP BY species.name; 
+-- List all Digimon owned by Jennifer Orwell.
+SELECT animals.name, owners.full_name  from animals  JOIN owners  ON (animals.owner_id=owners.id AND owners.full_name = 'Jennifer Orwell') JOIN species ON (animals.species_id=species.id AND species.name='Digimon'); 
+-- List all animals owned by Dean Winchester that haven't tried to escape.
+SELECT owners.full_name, animals.name FROM owners JOIN animals ON animals.owner_id=owners.id AND animals.escape_attempt = 0 WHERE owners.full_name = 'Dean Winchester';
+-- Who owns the most animals?
+SELECT owners.full_name , COUNT(animals.name) AS Animals_Count From owners JOIN animals ON owners.id=animals.owner_id GROUP BY owners.fullname ORDER BY Animals_Count DESC; 
```

Comment:
- As suggested by the previous reviewer, kindly fix the persisting error on line `100` on query.

## Guided suggestion:
`SELECT owners.full_name, COUNT(owners.full_name) FROM animals
   LEFT JOIN owners
   ON animals.species_id = owners.id
   GROUP BY owners.full_name
   ORDER BY COUNT(owners.full_name) DESC`